ext/hal/README

```diff
@@ -7,53 +7,3 @@ atmel
 The Atmel Software Framework (ASF) provides device header files and peripheral
```

Comment:
Why keep atmel, seems like you can remove this file.  since it doesn't mention altera.

Reply:
ok, will remove it

Reply:
done